action.yaml

```diff
@@ -43,14 +39,13 @@ inputs:
     description: 'Run `buf push`'
     required: false
     default: 'false'
+  commit-message:
+    description: 'commit message to use when pushing generated code'
+    required: false
+    default: '[skip ci] buf generated code from protos'
```

Comment:
I don't think this is conventional commit friendly. Should probably do like `chore` or something on it

Reply:
updated